.changelogs/export-special-chars.yml

```diff
@@ -0,0 +1,6 @@
+significance: patch
```

Comment:
We're introducing an abstract method here, which means that all the classes that extend `LLMS_Abstract_Exportable_Admin_Table` must now implement that otherwise we have a fatal. This is a breaking change, isn't it?

Reply:
well... I added it because it's a bug that it didn't already exist since other methods in the class reference the method already.

It's possible that other classes extending this didn't happen to use the method(s) that reference the missing abstract method.

You're right though. it's a breaking change this way.

If we added it as a stub with an empty string return it would fix the bug I'm seeing without making any breaking changes for other classes, right?

Reply:
yeah with a warning yeah

Reply:
like a doing it wrong. E.g.
https://github.com/gocodebox/lifterlms-rest/blob/trunk/includes/abstracts/class-llms-rest-controller-stubs.php#L90

Reply:
@eri-trabiccolo updated with `_doing_it_wrong()` per your suggestion